src/Storefront/Controller/ConsignmentController.php

```diff
@@ -63,33 +64,49 @@ class ConsignmentController extends StorefrontController
     /**
      * @var ConsignmentService
      */
-    private $consignmentService;
+    private ConsignmentService $consignmentService;
+
+    /**
+     * @var \MyPa\Shopware\Service\Order\OrderService
+     */
+    private OrderService $orderService;
 
     /**
      * @var ShipmentService
      */
-    private $shipmentService;
+    private ShipmentService $shipmentService;
+
+    /**
+     * @var \MyPa\Shopware\Service\ShippingOptions\ShippingOptionsService
+     */
+    private ShippingOptionsService $shippingOptionsService;
 
     /**
      * @var SystemConfigService
      */
-    private $systemConfigService;
+    private SystemConfigService $systemConfigService;
 
     /**
      * ConsignmentController constructor.
      *
-     * @param ConsignmentService  $consignmentService
-     * @param ShipmentService     $shipmentService
-     * @param SystemConfigService $systemConfigService
+     * @param  ConsignmentService     $consignmentService
+     * @param  ShipmentService        $shipmentService
+     * @param  OrderService           $orderService
+     * @param  ShippingOptionsService $shippingOptionsService
+     * @param  SystemConfigService    $systemConfigService
      */
     public function __construct(
-        ConsignmentService  $consignmentService,
-        ShipmentService     $shipmentService,
-        SystemConfigService $systemConfigService
+        ConsignmentService     $consignmentService,
+        ShipmentService        $shipmentService,
+        OrderService           $orderService,
+        ShippingOptionsService $shippingOptionsService,
+        SystemConfigService    $systemConfigService
     )
     {
         $this->consignmentService = $consignmentService;
         $this->shipmentService = $shipmentService;
+        $this->orderService = $orderService;
```

Comment:
Zie ik het nou verkeerd of wordt die orderservice die nieuw is aangemaakt nergens gebruikt?